create-mysql-db.sh

```diff
@@ -18,15 +18,15 @@ MYSQL=`which mysql`
 if [ -z "$1" ]
 then
   Q1="CREATE DATABASE IF NOT EXISTS $MYSQL_NAME;"
-  Q2="GRANT ALL ON *.* TO '$MYSQL_USER'@'$MYSQL_HOST' IDENTIFIED BY '';"
+  Q2="CREATE USER IF NOT EXISTS '$MYSQL_USER'@'$MYSQL_HOST'; GRANT ALL ON *.* TO '$MYSQL_USER'@'$MYSQL_HOST' IDENTIFIED BY '';"
```

Comment:
This is minor, but would it make sense to create a new line for this, so that "GRANT ALL" ends up as "Q3" and so forth.  You'd have to update the last line SQL="${Q1}${Q2}${Q3}${Q4}" but I think that'd be more readable and follow existing convention within this file.

Reply:
👍